src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -70,7 +70,6 @@
       <DesignTime>True</DesignTime>
       <DependentUpon>SqmOleCommandTarget.tt</DependentUpon>
     </Compile>
-    <Compile Include="Telemetry\SonarLintIntegrationPackage.IOleCommandTarget.cs" />
```

Comment:
That shouldn't have been a problem: calling a command before a package is loaded is normal. VS will load the package that is registered as providing the command if the package isn't already loaded.